### PR TITLE
added new abstracts, fields, and types.

### PR DESCRIPTION
**Pull request by hhunterr71**
added new abstracts, fields, and types.

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 1]
```diff
@@ -611,6 +611,50 @@ VAV_DSP_PDSCV:
...
+
+VAV_PDSCV_WDPC:
+  description: "Supply control VAV with damper percentage monitoring, and differential pressure control."
+  is_canonical: true
+  implements:
+  - VAV
+  - PDSCV
+  - WDPC
+
+VAV_SD_DTM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:650]: does this VAV not have zone temp sensor / setpoints?

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 2]
```diff
@@ -611,6 +611,50 @@ VAV_DSP_PDSCV:
...
+VAV_DSP_PDSCV_WDPC:
+  description: "Supply control VAV with dual setpoint damper percentage monitoring, and differential pressure control."
+  is_canonical: true
+  implements:
+  - VAV
+  - DSP
+  - PDSCV
+  - WDPC
+
+VAV_PDSCV_WDPC:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:642]: No temp control?

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 3]
```diff
@@ -611,6 +611,50 @@ VAV_DSP_PDSCV:
   - DSP
   - PDSCV
 
+VAV_CSP_DFVSC_DAIDC:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:614]: this VAV doesn't have damper percentage control?

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -509,6 +511,10 @@ VSFC:
   guid: "8dcdc4b0-af94-4468-8346-d28442447f84"
   description: "Variable speed control or monitor in frequency"
   is_abstract: true
+  opt_uses:
+  - run_status
```
[the remark this answers is not in the document]
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:518]: Removed

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 5]
```diff
@@ -7387,3 +7413,18 @@ NO2C2XDSP:
...
   - CONTROL
+
+PHWRMSC:
+  guid: "ef91c1f1-cc85-4690-b4dd-69a4678f7cb3"
+  description: "Potable hot water with return monitoring and supply water control."
+  is_abstract: true
+  opt_uses:
+  - low_potable_supply_water_temperature_alarm
+  uses:
+  - potable_hot_return_water_temperature_sensor
```
[the remark this answers is not in the document]
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7552]: Moved

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 6]
```diff
@@ -1841,3 +1841,25 @@ FCU_DFVSC_RTM_CO2M_HWZTC_CHWZTC:
   - HWZTC
   - CHWZTC
 
+FCU_DFSS_CSP_DXZC_RAIDC:
+  description: "Typical DX FCU with cooling-only (discharge control), and return air isolation damper control."
```
[the remark this answers is not in the document]
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:1845]: updated to - Typical DX FCU with single cooling setpoint control, and return air isolation damper control.

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 7]
```diff
@@ -802,6 +802,17 @@ MAU_NONCANONICAL_4:
   - supply_fan_run_status
   - supply_fan_speed_percentage_command
 
+MAU_DF2XSS_DFVSC2X_CHWZC_HWZC_VOADM:
+  description: "Multi-zone MAU with chilled water and heating water control and outside air monitoring."
+  is_canonical: true
+  implements:
+  - MAU
+  - DF2XSS
```
[the remark this answers is not in the document]
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml:810]: removed from one and added to the other. Updated MAU

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml (hunk 3 above)
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/VAV.yaml:614]: It does not.

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml (hunk 2 above)
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/VAV.yaml:642]: Confirmed no temp control.

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml (hunk 1 above)
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/VAV.yaml:650]: Confirmed, no zone temp sensor or septpoints

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 8]
```diff
@@ -746,7 +750,10 @@ DFSS2X:
   description: "Two discharge fan run command and status (start/stop). With dampers"
   is_abstract: true
   implements:
+  opt_uses:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:753]: The alignment got messed up here - please fix.

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 9]
```diff
@@ -802,6 +802,17 @@ MAU_NONCANONICAL_4:
   - supply_fan_run_status
   - supply_fan_speed_percentage_command
 
+MAU_DFSS2X_DFVSC2X_CHWZC_HWZC_VOADM:
+  description: "Multi-zone MAU with chilled water and heating water control and outside air monitoring."
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:806]: The description says multi-zone but you are using discharge abstracts

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 10]
```diff
@@ -7516,6 +7541,19 @@ NO2C2XDSP:
   implements:
   - CONTROL
 
+PHWRMSC:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7544]: Rename `POTHWSC`

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/PMP.yaml [hunk 11]
```diff
@@ -181,3 +181,11 @@ PMP_SS_VSC_NONCANONICAL_2:
   - mixing_valve_percentage_command
   - chilled_supply_water_temperature_setpoint
   - chilled_supply_water_temperature_sensor
+
+PMP_SS_SWTM_RWTM:
```
[the remark this answers is not in the document]
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/PMP.yaml]: Added DHW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 8 above)
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:753]: changed

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml (hunk 9 above)
Comment (hhunterr71) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml:806]: Single

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**